src/useVisibilityObserver/useVisibilityObserver.ts

```diff
@@ -0,0 +1,24 @@
+import { onMounted, ref, Ref } from 'vue'
+import { useIntersectionObserver } from '@/useIntersectionObserver'
+
+export function useVisibilityObserver(element: Ref<HTMLElement | undefined>): Ref<boolean> {
```

Comment:
We should expose the `options` from `useIntersectionObserver` here as an optional second argument. So a developer can customize the root, threshold, etc.